docs/core/contributing.md

```diff
@@ -5,7 +5,7 @@ title: Contribution Guide
 
 Our goal is to make contributing to the Libra project easy and transparent.
 
-<blockquote class="block_note">
+<blockquote className="block_note">
```

Comment:
This was just to quiet the warnings but we will most likely remove the classNames and apply a typography stylesheet.